public/app/features/alerting/unified/components/receivers/form/ChannelSubForm.tsx

```diff
@@ -53,6 +53,7 @@ export function ChannelSubForm<R extends ChannelValues>({
 
   const { control, watch, register, trigger, formState, setValue } = useFormContext();
   const selectedType = watch(fieldName('type')) ?? defaultValues.type; // nope, setting "default" does not work at all.
+  const parse_mode = watch(fieldName('settings.parse_mode')) ?? defaultValues.settings.parse_mode;
```

Comment:
Can `settings` be an empty object? I don't think we get type hinting here (ie. everything is `any`, right?)

Should we use optional chaining?

Reply:
good point